src/core/default_appearance.js

```diff
@@ -87,6 +87,94 @@ function parseDefaultAppearance(str) {
   return new DefaultAppearanceEvaluator(str).parse();
 }
 
+class AppearanceEvaluator extends EvaluatorPreprocessor {
+  constructor(str) {
+    super(str);
+    this.str = str;
+  }
+
+  parse() {
+    const operation = {
+      fn: 0,
+      args: [],
+    };
+    const result = {
+      fontSize: 0,
+      fontName: "",
+      fontColor: /* black = */ new Uint8ClampedArray(3),
+    };
+
+    let isInTextSection = false;
```

Comment:
Given that consistency is nice, please let's use the same name as we do in the regular evaluator:
```suggestion
    let parsingText = false;
```